src/dashboard/category-results-screen.js

```diff
@@ -4,16 +4,76 @@ import CategoryCard from './category-card.js';
 import { sub } from 'date-fns';
 
 function Paper ({ paper }) {
-  return h('div', { class: 'paper' }, [
+  const corrAuthor = findCorrespondingAuthor(paper).toString();
+
+  const paperElements = [
     h('a', { href: paper.finalURL, target: '_blank', class: 'paper-link' }, [
       h('div', { class: 'paper-read' }),
       h('div', { class: 'paper-title' }, paper.title)
     ]),
-    h('div', { class: 'paper-authors' }, paper.authors),
+    h('div', { class: 'paper-authors' }, finalizeAuthorsList(paper.authors, corrAuthor), [
```

Comment:
Does the order of the authors matter if we're planning to condense the author list? Currently I've set it up so the corresponding author with their institution tooltip is just tacked onto the end of the author list.

Reply:
The order of the authors is important. Typically first (few) author(s) performed the day to day research; last (few) author(s) usually reflect the supervisor, lab head etc.